functions/src/FirebaseFunctions.cs

```diff
@@ -221,6 +221,26 @@ public sealed class FirebaseFunctions {
       return new HttpsCallableReference(this, functionsInternal.GetHttpsCallable(name));
     }
 
+    /// <summary>
+    ///   Creates a
```

Comment:
hmmm this seems an extra line, can the reference fit in the same line?

Reply:
I copied it from above, but you are right, there isn't a need for it to be so broken up. Fixed the above one as well.